### DISCUSSION TITLE
How to remove custom domain from GitHub pages

### DISCUSSION THREAD

**Discussion by georgealexiou** [community/community#23498] [Other Feature Feedback, Questions, & Ideas]
<p>I’m trying to create a personal website for my portfolio to post on <a href="http://github.io" rel="nofollow noopener">github.io</a>. The problem is that I added a custom domain i bought a while back that I forgot I mapped to a different provider. I now want to remove this custom domain but everything I do doesn’t let me delete it. The repository I’m using is <a href="https://github.com/georgealexiou/georgealexiou.github.io" rel="nofollow noopener">https://github.com/georgealexiou/georgealexiou.github.io</a>.</p>
<p>I have checked other forums that said that I had to remove a CNAME file but I can’t find anything like that. </p>
<p>Thank you!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can follow the instructions in our help documentation for <a href="https://help.github.com/en/articles/adding-or-removing-a-custom-domain-for-your-github-pages-site" rel="nofollow noopener">adding and removing custom subdomains</a>.</p>
<p>Let us know if you have more questions.</p>

**Comment by ChandlerKilpatrick1** [comment]
<p><a class="mention" href="/lee-dohm">@lee-dohm</a> there does not appear to be docs on removing a custom domain anymore. </p>

**Comment by Adefunke** [comment]
<p><a class="mention" href="/chandlerkilpatrick1">@chandlerkilpatrick1</a> It’s there. You would have to clear your cache. <a href="https://help.github.com/en/github/working-with-github-pages/troubleshooting-custom-domains-and-github-pages" rel="nofollow noopener">https://help.github.com/en/github/working-with-github-pages/troubleshooting-custom-domains-and-github-pages</a></p>

**Comment by lordscarlet** [comment]
<p>It really isn’t there.</p>

**Comment by SharifPoetra** [comment]
<p>Isn’t there for me too</p>

**Comment by kevadsett** [comment]
<p>This is likely the reason: <a href="https://github.com/isaacs/github/issues/1213#issuecomment-381864035" rel="nofollow noopener">https://github.com/isaacs/github/issues/1213#issuecomment-381864035</a></p>

**Comment by Powerisinschool** [comment]
<p>Try this link: <a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#removing-a-custom-domain" class="inline-onebox" rel="noopener nofollow ugc">Managing a custom domain for your GitHub Pages site - GitHub Docs</a></p>

**Reply by Omobayode6** [reply]
It is there in this link. Thanks

**Comment by BishalN** [comment]
In my case I wanted to use github pages for portfolio site at bishaln.github.io
The problem was I had another repo that had the custom domain name that i wanted to remove, So i removed it and it seems to work
you can only remove the domain from the repo where you assigned it

**Comment by arbasil05** [comment]
I had a similar issue to, the problem with me was whenever i tried to open my default .io domain it automatically rerouted it to my custom domain, then i found a repository in my profile with the name of my default domain (like arbasil05.github.io) i have created it long ago when buying domain from name cheap using student developer pack but i forgot, so i went to the pages settings of that repo and removed the custom domain, then it worked fine

**Reply by arbasil05** [reply]
the CNAME file will be found inside that repo*
